### PR TITLE
2024-04-12 Marco Ieni

### PR DESCRIPTION
**Pull request by allenwyma**

### CODE REVIEW

File: _episodes/interviews/2024-04-12-marco-ieni.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+⁃ [@00:00] - Meet Marco Ieni, software engineer at Prisma, creator of release-plz
+⁃ [@01:17] - Release-plz creation & features
+⁃ [@08:50] - Conventional commit standard
+⁃ [@17:41] - Potential upcoming features
+⁃ [@21:25] - Gitea, Gitlab, Github integration
+⁃ [@28:14] - Release-plz development
+⁃ [@31:24] - Windows support
+⁃ [@36:22] - Message from Marco
+⁃ [@37:07] - Marco’s Rust podcast [RustShip](https://www.marcoieni.com/2023/08/rustship-a-new-rust-podcast/)
```
Comment (jonhoo) [inline _episodes/interviews/2024-04-12-marco-ieni.md:31]: This looks like an incorrect apostrophe character (should be a regular ')

File: _episodes/interviews/2024-04-12-marco-ieni.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+
+⁃ [@00:00] - Meet Marco Ieni, software engineer at Prisma, creator of release-plz
```
Comment (jonhoo) [inline _episodes/interviews/2024-04-12-marco-ieni.md:23]: Link to prisma?

File: _episodes/interviews/2024-04-12-marco-ieni.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Pavex with Marco Ieni"
```
Comment (jonhoo) [inline _episodes/interviews/2024-04-12-marco-ieni.md:2]: This doesn't look right.

**Review by jonhoo — CHANGES_REQUESTED** [review]

**Review by jonhoo — APPROVED** [review]

File: _episodes/interviews/2024-04-12-marco-ieni.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+
+⁃ [@00:00] - Meet Marco Ieni, software engineer at [Prisma](https://www.prisma.io/), creator of release-plz
```
Comment (jonhoo) [inline _episodes/interviews/2024-04-12-marco-ieni.md]: Wrong kinds of dashes again?

File: _episodes/interviews/2024-04-12-marco-ieni.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
+---
+title: "relase-plz with Marco Ieni"
+date: 2024-04-12T07:06:00Z
```
Comment (jonhoo) [inline _episodes/interviews/2024-04-12-marco-ieni.md:3]:
```suggestion
date: 2024-04-12T12:19:00Z
```

**Review by jonhoo — APPROVED** [review]

File: _episodes/interviews/2024-04-12-marco-ieni.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
+---
+title: "relase-plz with Marco Ieni"
```
Comment (robjtede) [inline _episodes/interviews/2024-04-12-marco-ieni.md:2]: typo "relase" -> "release"

Comment (robjtede) [reply _episodes/interviews/2024-04-12-marco-ieni.md:2]: https://github.com/rustacean-station/rustacean-station.org/pull/256

**Pull request merged**